device/bgp/bgpupdatereplay.yaml

```diff
@@ -0,0 +1,33 @@
+components:
+  schemas:
+    Bgp.UpdateReplay:
+      description: >-
+        Ordered BGP Updates ( including both Advertise and Withdraws ) to be sent in the order given in the input to the peer after the BGP session is established.
+      type: object
+      properties:
+        updates:
```

Comment:
Is there any possibility of any other type(s) other than `updates` under `Bgp.UpdateReplay`? Otherwise, we might consider reducing one nesting level.

Reply:
Indeed I have kept it for future controls needed if any e.g. providing a name to the object to be able to use that as a key on some future TBD action. (e.g. replay by user trigger )